docs/appkit/overview.mdx

```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
 
-Some of the key features of the AppKit SDK include:
+#### Framework Agnostic
+Integrate effortlessly with popular frameworks like React, Vue, Svelte, vanilla JavaScript, and more.
+
+#### EIP-6963 Support
+Seamlessly connect browser extension wallets for an enhanced user experience. Learn more about **EIP-6963 [here](https://eips.ethereum.org/EIPS/eip-6963).**
+
+#### Email Wallets
+Enable users to connect to dapps using just their email address, ensuring a smooth onboarding process.
+
+#### On-Ramp
+Allow users to securely purchase crypto with fiat, directly within your app, supporting over 100 cryptocurrencies.
+
+#### Customizable UI
+Choose between light and dark modes, accent colors, or fully customize the interface with your own branding.
+
+#### Multi-Chain Ready
+Easily configure support for EVM and non-EVM chains, giving users access to multiple blockchain networks.
+
+#### Hundreds of Wallets
+Connect with popular wallets like MetaMask, Rainbow, Coinbase, and many more to maximize accessibility.
+
+#### Transaction History
+Display all user transactions directly in the account view for added transparency.
+
+#### Transaction History
+Display a comprehensive list of all user transactions directly in the account view.
+
+#### Swaps
+Simplify token swapping for users with AppKit Modal, allowing secure, in-app swaps with just one line of code.
+
+#### Email & Social Login
+Expand your app’s reach by enabling users to connect using their email or social accounts. Implement these features with minimal code and leverage Smart Account functionality for advanced capabilities.
+
+#### One-Click Auth
+Streamline user authentication with WalletConnect v2 integration. Users can sign in and connect to your app with just one tap, supporting multi-chain and multi-account signing.
+
+#### Smart Accounts
+Enhance security and user convenience with features like multi-signature authorization and automated transaction workflows.
+
+#### Solana Support
+Bring the Solana blockchain to your application with simple, secure wallet connections and seamless transaction support.
+
+#### Notifications
+Engage your users with web3-native notifications sent straight to their wallets or through in-app and Web3Inbox notifications.
 
-- **Support for multiple frameworks.** Easily integrate with React, Vue, Svelte, vanilla JavaScript, and more.
-- **[EIP-6963.](https://eips.ethereum.org/EIPS/eip-6963) support** for browser extension wallets.
-- **Email Wallets.** Allow users to seamlessly connect to dapps with their email address.
-- **On-Ramp.** Enable users to buy crypto with fiat currency.
-- **Customizable UI**. Choose between light and dark modes and accent colors, or even infuse it with your own branding.
-- **Multi-chain ready.** Easily set up support for any EVM chain.
-- **Hundreds of Wallets.** Connect with MetaMask, Rainbow, Coinbase and many more.
-- **Transaction History.** View all transactions from the account view.
+#### Telegram MiniApps
```

Comment:
we are not supporting social logins with tg mini apps ... or we are ?

Reply:
We support social logins